src/components/Navbar/SideWidgets/FilterSelect.vue

```diff
@@ -72,6 +93,8 @@ function selectAllTrackers() {
         <template v-slot:prepend-item>
           <v-list-item :title="$t('common.disable')" @click="selectAllStatuses" />
           <v-list-item :title="$t('common.active')" @click="selectActive" />
+          <v-list-item :title="$t('common.error')" @click="selectError" />
+          <v-list-item :title="$t('common.offline')" @click="selectOffline" />
```

Comment:
It may be a good idea to separate those 3 keys if we keep growing it as they are only used there.

I suggest:
- `navbar.side.filters.state.active`
- `navbar.side.filters.state.error`
- `navbar.side.filters.state.offline`

And so the existing key `navbar.side.filters.state` should be moved to `navbar.side.filters.state.title`